repository_service_tuf/cli/admin/metadata.py

```diff
@@ -724,21 +752,33 @@ def sign(context, api_url: Optional[str]) -> None:
     pending_roles = _get_pending_roles(settings, api_url)
     role_info: MetadataInfo
     rolename: str
+
+    msg: str
+    if not delete:
+        msg = "sign"
+    else:
+        msg = "delete signing process"
+
     while True:
         rolename = prompt.Prompt.ask(
-            "\nChoose a metadata to sign",
+            f"\nChoose a metadata to {msg}",
             choices=[role for role in pending_roles],
         )
         role_info = MetadataInfo(
             Metadata.from_dict(copy.deepcopy(pending_roles[rolename]))
         )
         _print_md_info(role_info, False)
         confirmation = prompt.Confirm.ask(
-            f"\nDo you still want to sign {rolename}?"
+            f"\nDo you still want to {msg} {rolename}?"
         )
         if confirmation:
             break
 
+    if delete:
+        _delete_signing_process(role_info.type, settings, api_url)
+        console.print("\nSigning process deleted!\n")
+        sys.exit()
```

Comment:
The delete process is asynchronous (as all requests in RSTUF API).
It would be good to return the task id. Another option is showing the task status (as in the bootstrap).